inc/az_ulib_ipc_api.h

```diff
@@ -104,10 +104,23 @@ static inline AZ_ULIB_RESULT az_ulib_ipc_deinit(void) {
 /**
  * @brief   Publish a new interface on the IPC.
  *
+ * This API publishes a new interface in the IPC using the interface descriptor. The interface
+ * descriptor shall be valid up to the interface is unpublished with success.
```

Comment:
>  [](start = 34, length = 1)

lets add something like "the point when" here. 
So sentence being "... shall be valid up to the point when the interface ..."